code/modules/surgery/organs/stomach.dm

```diff
@@ -156,6 +168,7 @@
 	name = "digestive crystal"
 	icon_state = "stomach-p"
 	desc = "A strange crystal that is responsible for metabolizing the unseen energy force that feeds plasmamen."
+	metabolism_efficiency = 0.08
```

Comment:
I don't particularly care, but we do have a policy about changes to non-human races.  Giving plasmamen specifically a higher digestion rate might matter.

Reply:
I do try to spread some flavor around. Also they have a hard time eating without taking the helmet off.